cloudconfig/podcfg/podcfg.go

```diff
@@ -267,7 +289,9 @@ func (cfg *ControllerConfig) VerifyConfig() error {
 // always needed.
 func NewControllerPodConfig(
 	controllerTag names.ControllerTag,
-	machineID, series string,
+	machineID,
```

Comment:
let's call this podID or something?
also podNonce

Reply:
changed